packages/terra-modal-manager/tests/wdio/modal-manager-spec.js

```diff
@@ -313,10 +313,16 @@ Terra.describeViewports('ModalManager - Behaviors', ['large'], () => {
     });
 
     describe('Modal Focus', () => {
-      it('focuses on the modal when opened', () => {
+      it('is on the first element when modal when opened', () => {
         $('#root-component .disclose-small').click();
         $('[class*="slide-group"] #DemoContainer-1 .maximize').waitForDisplayed({ timeout: 1000 });
-        expect($('[aria-modal="true"][role="dialog"]').isFocused()).toEqual(true);
+
+        const firstElementWithModalBeginAttribute = $('[aria-modal="true"][role="dialog"] [data-terra-abstract-modal-begin="true"]');
+        expect(firstElementWithModalBeginAttribute.getText() === '').toEqual(true);
```

Comment:
Would something like this work?
```suggestion
        expect(firstElementWithModalBeginAttribute.getText()).toBe('');
```